transforms.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, size):
 
     def __call__(self, image, target):
         image = pad_if_smaller(image, self.size)
-        target = pad_if_smaller(target, self.size)
+        target = pad_if_smaller(target, self.size, fill=255)
```

Comment:
Padding area will be ignored by setting them to `255`.
`0` is background area and we want to learn them along with the target label's area.